src/main.rs

```diff
@@ -315,13 +335,97 @@ impl<T> MaybeExternal<T> {
     }
 }
 
+fn infer_language(module: &wasmbin::Module) -> Result<Language> {
+    let mut imports = vec![];
```

Comment:
Incredibly minor question:

`Vec::new()`

is used to create empty vectors elsewhere in the file. Is there an advantage to using the shortened syntax here?

Reply:
I think they are the same, but have changed for consistency